### ISSUE TITLE
Use of deprecated /events api

### ISSUE THREAD

**Issue by APwhitehat** [matrix-org/sytest#459]
`tests/50federation/31room-send.pl` relies on `/events` api which is deprecated.
sytest should use `/sync`.
https://matrix.org/docs/spec/client_server/r0.3.0.html#get-matrix-client-r0-events

**Comment by ara4n** [comment]
/events is still used when peeking rooms, but the hope is to replace it entirely with /sync in the near future - probably when [paginated /sync](https://github.com/matrix-org/synapse/pull/893) finally lands, as that gave a way to request /sync on arbitrary rooms (permission allowing).

**Comment by APwhitehat** [comment]
My concern lies on the fact that sytest uses `/events` which makes dendrite fail many tests.
Which is not ideal because failure of the deprecated API means you cannot test other APIs.

**Comment by ara4n** [comment]
yup, understood. i think the answer would be to mark those tests as non-critical for now, given /events is on the way out.
